aegis-web/yo/app/views/search.html

```diff
@@ -73,26 +73,37 @@
   </nav>
   <section id="filterdropdown" class="collapse search-result-filter">
     <div class="row result-filter-body">
-      <li class="col-md-4">
+        <li class="col-md-3">
         <ul>
-          <li class="dropdown-header">Type</li>
-          <li><a class="filter-menu-uppercase"
+        <div class="dropdown">
+                <p>Type
+                    <button class="btn btn-default dropdown-toggle" type="button" id="dropdownType">
+                        
+                        <span class="caret"></span>
+                    </button>
+                </p>
```

Comment:
Have a look at your local styleguide.
Remove `<p>` keeping only the `<button>`. 
Put your text "Type" before `<span>` inside the `<button>`.
add  `data-toggle="dropdown"` to your `<button>`

Reply:
ok, done